src/shared/components/Header/Header.test.tsx

```diff
@@ -31,19 +31,4 @@ describe('Header', () => {
       screen.getByText('Title is optional', { selector: 'a' })
```

Comment:
Did you set the testing language to dutch but then this should be in dutch too right?

Reply:
No, this is just a dummy string to the test that a Header can receive an optional title.